nansat/tests/test_domain.py

```diff
@@ -11,7 +11,7 @@
 #               http://www.gnu.org/licenses/gpl-3.0.html
 #------------------------------------------------------------------------------
 import unittest
-import os
+import os, sys
```

Comment:
Separate between two lines.

pep8 says:

Imports should usually be on separate lines, e.g.:

Yes: import os
     import sys

No:  import sys, os